desktop-styles.css

```diff
@@ -0,0 +1,480 @@
+  line-height: 24px;
+  text-transform: uppercase;
+  font-weight: 600;
+  font-size: 18px;
+  color: #344563;
+}
+
+.content-container > p {
+  padding: 0;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  max-width: 400px;
+  margin: 24px;
+}
+
+.content-container > .technologies {
+  list-style-type: none;
+  padding: 3px 0;
+  margin: 0 24px 56px 24px;
+  display: flex;
+}
+
+.content-container > .technologies > li {
+  padding: 4px 12px;
+  margin-right: 8px;
+  background: #ebebff;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+#button-container {
+  padding: 12px 0;
+  width: 100%;
+}
+
+.btn {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  padding: 12px;
+  box-sizing: border-box;
+  color: #396df2;
+  border: 1px solid #6070ff;
+  background: #fff;
+  border-radius: 8px;
+  width: max-content;
+  margin-left: 24px;
+  cursor: pointer;
+}
+
+.btn > a {
+  color: inherit;
+  text-decoration: none;
+}
+
+.btn:hover {
+  background: #6070ff;
+  color: #fff;
+  transform: scale(1.2);
+}
+
+.btn:active {
+  background: #424daa;
+  color: #fff;
+  transform: scale(1.2);
+  transform: rotate(-5deg);
+}
+
+#about-container {
+  margin-top: 0;
+  display: flex;
+  justify-content: flex-start;
+  background: #fff;
+  border-radius: 0 90px 0 0;
+  box-sizing: border-box;
+  width: 100%;
+}
+
+#about {
+  margin-left: 142px;
+  flex-direction: column;
+  justify-content: flex-start;
+  margin-bottom: 0 0 100px 0;
+  list-style-type: none;
+  padding: 0;
+  display: flex;
+  width: 350px;
+}
+
+.about-container p {
+  margin-top: 10px;
+}
+
+.catagories-section {
+  list-style: none;
+  margin-top: 40px;
+  margin-left: 30px;
+  width: 50%;
+}
+
+#categories {
+  flex-direction: column;
+  justify-content: flex-start;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  align-items: flex-start;
+}
+
+#item-details {
+  display: flex;
+  justify-content: flex-start;
+  width: 200px;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+#categories-item {
+  line-height: 24px;
+  display: flex;
+  justify-content: space-between;
+  color: #253858;
+  padding: 24px 12px;
+  box-sizing: border-box;
+  width: 80%;
+  min-height: 72px;
+  font-weight: 500;
+  font-size: 20px;
+  margin-top: 20px;
+}
+
+#categories-item > .fa-solid {
+  color: #727e93;
+}
+
+.bottom-border {
+  border-bottom: 1px solid #e3e5e9;
+}
+
+.detail {
+  background: #f7f7f9;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 10px;
+  box-sizing: border-box;
+  height: 150px;
+  color: #253858;
+}
+
+.detail > img {
+  mix-blend-mode: multiply;
+  padding-right: 16px;
+  object-fit: contain;
+}
+
+footer {
+  width: 100%;
+  height: 779px;
+  background-image: url(./images/desktop-background-footer.svg);
```

Comment:
- [ **OPTIONAL** ] APPLICABLE FOR DESKTOP VERSION: The background image is repeating making the UI broken. I strongly advise you to kindly fix the image issue to enhance the UI. 👍🏽  

Your design 👇🏽 
<img width="1440" alt="Screenshot 2022-12-10 at 1 14 57 AM" src="https://user-images.githubusercontent.com/24832846/206790020-104e6bf6-0677-441f-a48c-835f5f2a99f8.png">

- [ **OPTIONAL** ] APPLICABLE FOR MOBILE VERSION: Your contact section for the mobile version seems to be broken. I would again suggest you to please consider fixing this issue. You will be using this portfolio for job application purposes down the road. 👍🏽 💯 😄 

Your design 👇🏽 
<img width="377" alt="Screenshot 2022-12-10 at 2 01 48 AM" src="https://user-images.githubusercontent.com/24832846/206791546-64d96e55-d13f-4e99-867d-04d756d32904.png">


Figma design 👇🏽 
<img width="340" alt="Screenshot 2022-12-10 at 2 04 56 AM" src="https://user-images.githubusercontent.com/24832846/206791572-3a68d318-fcb3-47c1-b838-9aad3733db4d.png">